Martium.TravelInfo/Forms/TravelInfoForm.cs

```diff
@@ -9,22 +13,26 @@ namespace Martium.TravelInfo.Forms
     public partial class TravelInfoForm : Form
     {
         private readonly TravelInfoRepository _travelInfoRepository;
+        private TravelInfoSettingsModel _travelInfoSettingsModel;
+
 
         public TravelInfoForm()
         {
             _travelInfoRepository = new TravelInfoRepository();
+            _travelInfoSettingsModel = new TravelInfoSettingsModel();
```

Comment:
For models initialization is not required, only for classes which has methods so you can remove this line